docs/explore-command-design.md

```
@@ -0,0 +1,178 @@
+  --region <slug>    Restrict to one region (default: all regions)
+  --format <fmt>     json | table   (default: json)
+```
+
+### `explore regions`
+```
+partiful explore regions [--format json|table]
+  Lists region slugs + the tag list (id + friendly name) for --tag filtering.
+```
+
+### `explore rsvp <eventId>`
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)